src/App.js

```diff
@@ -1,23 +1,31 @@
 import React from "react";
 import "./App.css";
 import IconContainer from "./components/IconContainer";
-import { Switch, Route } from "react-router-dom";
+import { BrowserRouter, Switch, Route } from "react-router-dom";
 import Welcome from "./components/Welcome";
 import Error404 from "./components/Error404";
+import KanbanBoard from "./container components/KanbanBoard";
+import NavBar from "./components/Navbar";
 import ApplicantDetailPage from "./components/ApplicantDetailPage";
+import ListView from "./container components/ListView";
```

Comment:
Is there an period (space) between the **container** and **component**? Maybe it is better to use _underline_ or _camelcase convention_.

Reply:
I would use two folder but I wait for @balanza answer.

Reply:
I agree on that:
`src/components` for dumb components (or presentationals)
`src/containers` for smart components (or containers)

Reply:
Great. I had created _containers_ folder at the beginning of the project when I had created the _Skeleton_ branch but it seems, it is not added to the project because it was empty so now I added a _containers_ folder to the project and put the _ApplicantDetailPageContainer_ inside it.
From now on we can use it for the _containers components_.